content/hi/docs/reference/glossary/containerd.md

```diff
@@ -0,0 +1,18 @@
+---
+title: कंटेनरडी (containerd)
+id: containerd
+date: 2019-05-14
+full_link: https://containerd.io/docs/
+short_description: >
+  सादगी, मजबूती और सुवाह्यता पर जोर देने वाला कंटेनर रनटाइम
+
+aka:
+tags:
+  - tool
+---
+
+सादगी, मजबूती और सुवाह्यता पर जोर देने वाला कंटेनर रनटाइम
+
+<!--more-->
+
+कंटेनरडी एक {{< glossary_tooltip text="कंटेनर" term_id="container" >}} रनटाइम है जो Linux या Windows पर एक डैमन के रूप में चलता है। कंटेनरडी कंटेनर इमेजेस को लाने और संग्रहीत करने, कंटेनरों को निष्पादित करने, नेटवर्क एक्सेस प्रदान करने, आदि का ध्यान रखता है।
```

Comment:
```suggestion
कंटेनरडी एक {{< glossary_tooltip text="कंटेनर" term_id="container" >}} रनटाइम है जो Linux या Windows पर एक डैमन के रूप में चलता है। कंटेनर डी कंटेनर इमेजेस को लाने और संग्रहीत करने, कंटेनरों को निष्पादित करने, नेटवर्क एक्सेस प्रदान करने, आदि का ध्यान रखता है।
```
Native Hindi speakers might pronounce it in a different way (containerd) , hence suggesting this. Your thoughts about it?

Reply:
I don't think the spacing makes a difference. @bishal7679 : Please weigh in here.